pkg/k8s/aggregator.go

```diff
@@ -60,10 +63,14 @@ func (i *IngressAggregator) OnUpdate(old, new interface{}) {
 }
 
 //AddSource adds a new source for watching ingresses, must be called before running
-func (i *IngressAggregator) AddSource(source cache.ListerWatcher) {
+func (i *IngressAggregator) AddSource(source cache.ListerWatcher, endpoint []string) {
 	//Todo implement handler for events
 	store, controller := cache.NewIndexerInformer(source, &v1.Ingress{}, time.Minute, i, cache.Indexers{})
-	i.stores = append(i.stores, store)
+	cachestore := Store{
+		cachestore: store,
+		endpoints:  endpoint,
```

Comment:
```suggestion
		endpoints:  endpoints,
```
Just `endpoint` sounds like we putting a single endpoint into the Store's `endpoints` array, which can be confusing.

Reply:
Good suggestion 
I replace it with endpoints